README.md

```diff
@@ -15,3 +16,47 @@
 
 ### Run unit tests
     yarn test:unit
+
+## Using this library
+
+Package requirements, to be added after `vue create ...`
+
+```bash
+# Vuetify
+vue add vuetify
+
+# FontAwesome
+yarn add @fortawesome/fontawesome-free -D
+# OR add the CDN link in index.html
+# <link href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" rel="stylesheet">
+```
+
+To use components in your Vue.js app, the Vue instance must meet some requirements.  The basic requirements are:
+
+```javascript
+Vue.use(Vuetify);
```

Comment:
This part is automatically handled as a side effect of importing components from index.js, so we can omit it from here.